assets/mechanics.js

```diff
@@ -25,13 +26,15 @@ const ball = {
 
 class Obstacle {
   constructor(x, y, width, height, padding, color) {
+    this.floor_id = floor_id;
+    floor_id +=1;
     this.floorX = x;
     this.floorY = y;
     this.floorWidth = width;
     this.floorHeight = height;
     this.padding = padding;
     this.floorcolor = color;
-
+    this.deleted = false;
```

Comment:
The idea is to have 'active' and 'inactive' ledges. 
A ledge is initially active (i.e not deleted)